pallets/delegation/src/lib.rs

```diff
@@ -179,7 +179,6 @@ pub mod pallet {
 	}
 
 	#[pallet::pallet]
-	#[pallet::generate_store(pub(super) trait Store)]
```

Comment:
The storage getter was also deprecated?

Reply:
Yes it is [PR](https://github.com/paritytech/substrate/pull/13535)

Reply:
Storage getters seem to still be in. The PR was partially fixing the ticket, so I guess the deprecation of the storage getters will come in a next release.